### ISSUE TITLE
Addresses Tab -> Freeze -> Remove 'Spend From' option for freezed addresses

### ISSUE THREAD

**Issue by Anilkumar18** [Feathercoin-Foundation/electrum-ftc#66]
## Expected Behavior
Addresses Tab -> Freeze -> Remove 'Spend From' option for freezed addresses

## Current Behavior
Addresses Tab -> Freeze -> Remove 'Spend From' option is available even for freezed addresses

## Steps to Reproduce
1.Open a Wallet
2.Go to Addresses tab
3.Right click on an address which has balance and Freeze it
4.Now again right click on same address and select 'Spend From' user is taken to Send tab but will not be able to send coin as its freezed.
5.Either 'Spend from' option should be removed for Freezed addresses when rt clicked or proper error message'Address Frozen' should be displayed.

## Context
Currently it takes the user to the send tab and only after the user enters an amount , the status bar displays a message "Not enough Funds "Amt" frozen

## Your Environment

* Version used: Electrum FTC 3.0.6.5
* Operating System and version: Windows 10

Note : In Bitcoin , user is able to send amount from a Freezed address

**Comment by ghost** [comment]
I think this issue needs to be closed, as this behavior got fixed in the upstream repo.
We might want to open a new issue, since when multiple adresses(frozen and unfrozen ones) are marked, the option won't be shown. 
Question is now following: Should the option be shown, if at least one address is unfrozen?

**Comment by lclc** [comment]
Thanks!

Please open an issue upstream @cmm-31 with a clear description of the problem (similar to the structure of this issue).
